src/ctia/http/server_service.clj

```diff
@@ -6,32 +6,35 @@
             [puppetlabs.trapperkeeper.core :as tk]
             [puppetlabs.trapperkeeper.services :refer [service-context]]))
 
-(defprotocol CTIAHTTPServerService)
+(defprotocol CTIAHTTPServerService
+  (get-graphql [this] "Returns an instance of graphql.GraphQL")
+  (get-port [this] "Returns the port bound by HTTP server"))
 
 (tk/defservice ctia-http-server-service
   CTIAHTTPServerService
   [HooksService
    StoreService
    IAuth
-   GraphQLService
    GraphQLNamedTypeRegistryService
    IEncryption
    ConfigService]
-  (start [this context] (core/start context
-                                    ((:get-in-config ConfigService) [:ctia :http])
-                                    {:ConfigService (-> ConfigService
-                                                        (select-keys [:get-config
-                                                                      :get-in-config]))
-                                     :HooksService (-> HooksService 
-                                                       (select-keys [:apply-hooks
-                                                                     :apply-event-hooks])
-                                                       hooks-svc/lift-hooks-service-fns)
-                                     :StoreService (-> StoreService 
-                                                       (select-keys [:read-store
-                                                                     :write-store])
-                                                       store-svc/lift-store-service-fns)
-                                     :IAuth IAuth
-                                     :GraphQLService GraphQLService
-                                     :GraphQLNamedTypeRegistryService GraphQLNamedTypeRegistryService
-                                     :IEncryption IEncryption}))
-  (stop [this context] (core/stop context)))
+  (start [_ context] (core/start context
```

Comment:
Can you add a newline after `[_ context]`?

Fairly handy to be able to distinguish between definitions and invocations at a glance.